src/edutap/wallet_google/protocols.py

```diff
@@ -6,6 +6,14 @@
 @runtime_checkable
 class ImageProvider(Protocol):
 
+    async def active(self) -> bool:
+        """
+        Indicates if the Protocol Service is active and should be used.
+
+        :returns: bool if service is configured and available.
+        """
+        return True  # should be changed to default False on next major Update
```

Comment:
in protocols you do not add any code here. its not an ABC.
this has no effect, since one does not inherit from protocols.